src/zope/error/error.py

```diff
@@ -225,8 +229,8 @@ def _do_copy_to_zlog(self, now, strtype, url, info):
             next_when += _rate_restrict_period
             _rate_restrict_pool[strtype] = next_when
             try:
-                six.reraise(info[0], info[1], info[2])
-            except:
+                reraise(info[0], info[1], info[2])
+            except BaseException: # Yes, we really want everything.
                 logger.exception(str(url))
```

Comment:
Good point.

It's *nearly* equivalent. The `reraise` version will have some extra stack frames in it, whereas the `error` version will not. Those extra stack frames are just confusing, IMO, so I don't see why we shouldn't switch.